### ISSUE TITLE
Properties Window does not pop up when clicking "Game" » Manage » Properties

### ISSUE THREAD

**Issue by ReShout876** [ValveSoftware/steam-for-linux#10270]
#### Your system information

* Steam client version: 1701289036
* Distribution (e.g. Ubuntu):  Bunsenlabs Linux (Debian 11)
* Opted into Steam client beta?: No
* Have you checked for system updates?: yes, all packages up-to-date
* Steam Logs:  [steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/13530913/steam-logs.tar.gz)
* GPU: AMD RX 6650 XT 8GB

#### Please describe your issue in as much detail as possible:
In short, I'd like to manage a game's properties. I'd like to make the game My Summer Car to use Proton Experimental. I click on Manage » Properties - the properties window doesn't appear/pop up. (this is a fresh install)

#### Steps for reproducing this issue:

1.  Select a downloaded game.
2. Click Manage.
3. Click Properties
[Video](https://streamable.com/6df0fa)

Extra information: the DE is openbox

**Comment by kisak-valve** [comment]
Hello @ReShout876, this issue is already being tracked at #9273.

You might be able to workaround this issue by holding a modifier key like control while clicking the dropdown menu. You can also try enabling Steam's settings -> `Interface` -> `Enable context menu focus compatibility mode`.

Closing as a duplicate.

**Comment by ReShout876** [comment]
Hello @kisak-valve 
Your advise worked. Thank you very much. Have a great day!

**Comment by sjukfan** [comment]
Got it here too in Steam version 1705108172
